requirements.prod.txt

```diff
@@ -0,0 +1,7 @@
+Click
```

Comment:
we should have only requirements.txt (with all the dependencies necessary to run the app) and eventually additional ones for dev (with pytest, pytest-bdd, coverage, sphinx, etc, tools necessary for development only)